crates/holochain_zome_types/src/block.rs

```diff
@@ -10,12 +10,114 @@ use rusqlite::types::ToSqlOutput;
 #[cfg(feature = "rusqlite")]
 use rusqlite::ToSql;
 
-// Everything required for a coordinator to block some agent on the same DNA.
+mod kitsune_p2p_block {
```

Comment:
These types probably never belonged in kitsune... I named this kitsune_p2p_block to minimize changes, but if you feel I should rename this now I'm happy to.

Reply:
We can come back to this when turning to warranting.

Reply:
*no-op, rather than "broken" exactly